python/cuml/dask/ensemble/randomforestclassifier.py

```diff
@@ -232,6 +233,7 @@ def __init__(
             rfs_wait.append(r)
 
         wait(rfs_wait)
+        raise_exception_from_futures(rfs_wait)
```

Comment:
It seems like this should be added after all wait calls in just about all the Dask code.

Reply:
Yeah, I was hoping to start with it in one place to get a manageable PR that could go into 0.10

Reply:
That's fine w/ me. We can do a separate PR for all the remaining algos in 0.11.